Jenkinsfile

```diff
@@ -0,0 +1,113 @@
+#!groovy
+
+// https://github.com/feedhenry/fh-pipeline-library
+@Library('fh-pipeline-library') _
+
+stage('Trust') {
+    enforceTrustedApproval('aerogear')
+}
+
+def operatorName = "unifiedpush-operator"
+def openshiftProjectName = "test-${operatorName}-${currentBuild.number}-${currentBuild.startTimeInMillis}"
+def operatorDockerImageName = "docker-registry.default.svc:5000/${openshiftProjectName}/${operatorName}-test:latest"
+def testFileName = 'go-test.sh'
+
+def testFileContent = """#!/bin/sh
+${operatorName}-test -test.parallel=1 -test.failfast -root=/ -kubeconfig=incluster -namespacedMan=namespaced.yaml -test.v
+"""
+
+def dockerfileContent = """
+FROM registry.access.redhat.com/ubi7-dev-preview/ubi-minimal:7.6
+
+COPY ${operatorName} /usr/local/bin/${operatorName}
+COPY build/bin /usr/local/bin
+RUN  /usr/local/bin/user_setup
+ENTRYPOINT ["/usr/local/bin/entrypoint"]
+USER 1001
+
+ADD ${operatorName}-test /usr/local/bin/${operatorName}-test
+ADD deploy/operator.yaml /namespaced.yaml
+ADD ${testFileName} /${testFileName}
+"""
+
+node ('operator-sdk') {
+    stage ('Checkout') {
+        checkout scm
+    }
+    stage('Vendor the dependencies') {
+        sh 'dep ensure'
+    }
+
+    openshift.withCluster('operators-test-cluster') {
+        generateKubeConfig()
+
+        stage('New project in OpenShift') {
+            openshift.newProject(openshiftProjectName)
+        }
+
+        openshift.withProject(openshiftProjectName) {
+            stage("Build ${operatorName} & ${operatorName}-test binaries") {
+                sh """
+                export GOOS=linux GOARCH=amd64 CGO_ENABLED=0
+                go build -o ${operatorName} cmd/manager/main.go
```

Comment:
Maybe this should execute `make code/compile`, so that it's the same as what we'd run locally?